code/game/gamemodes/game_mode.dm

```diff
@@ -42,8 +42,15 @@ var/global/list/additional_antag_types = list()
 	var/list/dead_players = list()
 	var/player_count = 0
 
+	//Distress call variables.
+	var/list/datum/emergency_call/all_calls = list() //initialized at round start and stores the datums.
```

Comment:
I don't believe DM actually supports typed lists
this varname is too ambiguous.
And we usually managed globals like baycode

Maybe do GLOBAL_LIST_EMPTY(all_distress_call_datums)

Reply:
This variable is only used in ERT and DM can still use list datum, so it is better to ignore the use of global variables.